frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -100,28 +110,65 @@ export const ClusterServiceVersionTableHeader = () => {
     },
     {
       title: 'Namespace',
-      sortField: 'metadata.namespace',
+      sortFunc: 'namespace',
       transforms: [sortable],
       props: { className: tableColumnClasses[1] },
     },
     {
-      title: 'Status',
+      title: 'Managed Namespaces',
+      sortFunc: 'managedNamespace',
+      transforms: [sortable, wrappable],
       props: { className: tableColumnClasses[2] },
     },
     {
-      title: 'Deployment',
-      sortField: 'spec.install.spec.deployments[0].name',
-      transforms: [sortable],
+      title: 'Status',
       props: { className: tableColumnClasses[3] },
     },
     {
-      title: 'Provided APIs',
+      title: 'Last Updated',
       props: { className: tableColumnClasses[4] },
     },
+    {
+      title: 'Provided APIs',
+      props: { className: tableColumnClasses[5] },
+    },
     {
       title: '',
+      props: { className: tableColumnClasses[6] },
+    },
+  ];
+};
+
+export const ClusterServiceVersionTableHeader = () => {
```

Comment:
nite: I have a tendency to want to deduplicate rather than copy & paste:

```typescript
const sharedFirst = [{title: 'Name'.....}]
const sharedLast = [{title: 'Status'...},{title: 'Last Updated'...},{title: 'Provided APIs'...},{title: ''...}]
const AllProjectsTableHeader = sharedFirst
  .concat([{title: 'Operator Namespace'}])
  .concat(sharedLast)
const SingleProjectTableHeader = sharedFirst
  .concat([{title: 'Operator Namespace'}, {title: 'Managed Namespace'}])
  .concat(sharedLast)
```
But I wouldn't block on it, I'm betting some prefer the simplicty of just listing them directly.

Reply:
However, this comment does reveal that `props: { className: tableColumnClasses[0,1,2,3,4......] },` may be broken if that was done.

Reply:
You may instead need a little helper func to give back either of these arrays, but ensure the `tableColumnClasses` list is correct for either table.

Reply:
The tableColumnClasses are intentional; we only want to display certain columns at certain breakpoints. I can refactor though.

Reply:
The first is 1,2,3,4,5,6. the other is 0,2,3,4,5,6.  
If the skip is intentional, then 👍 
Looked like the kind of thing that might be an oops :)

Reply:
I'm refactoring using some string-based classes; let me know if that's easier to read once you can see it (just working out a few linter kinks).